Modules/Mailer/Controllers/MailerController.php

```
@@ -29,124 +37,129 @@ public function __construct()
      *
      * @originalFile MailerController.php
      */
-    public function invoice($invoice_id)
+    public function invoice(Request $request, int $invoice_id)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)